File: from-1401-to-1500/1460-make-two-arrays-equal-by-reversing-subarrays/solution.py
class Solution(object):
    def canBeEqual(self, target, arr):
        """
        :type target: List[int]
        :type arr: List[int]
        :rtype: bool
        """
        from collections import defaultdict
        d_arr = defaultdict(int)
        d_tar = defaultdict(int)
        
        for item in arr:
            d_arr[item] += 1
            
        for item in target:
            d_tar[item] += 1
            
#         for key in d_arr.keys():
#             if key not in d_tar.keys() or d_tar[key] != d_arr[key]:
#                 return False
            
        return d_tar == d_arr
    
    
        

File: from-1401-to-1500/1460-make-two-arrays-equal-by-reversing-subarrays/solution.go
func canBeEqual(target []int, arr []int) bool {
    if len(arr) != len(target) {
        return false
    }
    sort.Ints(arr)
    sort.Ints(target)
    for i := 0; i < len(arr); i ++ {
        if arr[i] != target[i] {
            return false
        }
    }
    return true
}